### PR TITLE
feature/equality checker

### PR DESCRIPTION
**Pull request by davidhopkinson26**
Added EqualityChecker transformer to check it two columns are equal

### CODE REVIEW

**Comment by Sarah-TaylorKnight** [comment]
Can you update this to PRing into develop rather than main?

**Comment by davidhopkinson26** [comment]
> Can you update this to PRing into develop rather than main?

updated target branch

File: tubular/comparison.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
+from tubular.base import BaseTransformer
+import pandas as pd
+
+
+class EqualityChecker(BaseTransformer):
+
+    """Transformer to check if two columns are equal.
+
+    This should be used to check if two columns are equal.
```
Comment (TommyMatthews) [inline tubular/comparison.py:9]: Just saying the same thing twice here

Comment (davidhopkinson26) [reply tubular/comparison.py]: changed in: [82645c5](`https://github.com/lvgig/tubular/pull/55/commits/82645c54f664f406abb18540a6cb9ae8158ec3f9`)

**Review by TommyMatthews — APPROVED** [review]

**Pull request merged**